appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -5788,4 +5788,8 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   @DefaultMessage("Time")
   @Description("Set x-axis label values Type as Time")
   String labelTime();
+
+  @DefaultMessage("Welcome to App Inventor Neo! If you are looking for the classic App Inventor look, you can switch in the User Interface Settings, or <a href=\"\">click here</a>.")
```

Comment:
<a href="">click here</a> ??? This just reloads.

Reply:
> click here ??? This just reloads.

It worked for me...

The actual link is in the words "click here." I initially expected it to work if I clicked anywhere on the popup. I don't know if it's possible to do that, but it would be easier to understand.